### PR TITLE
Consolidate Cloud Native Events schema

### PR DESCRIPTION
**Pull request by jzding**

### CODE REVIEW

File: examples/producer/main.go [hunk 1]
```diff
@@ -92,7 +94,7 @@ RETRY:
 						Resource:  pub.Resource,
 						DataType:  cneevent.NOTIFICATION,
 						ValueType: cneevent.ENUMERATION,
-						Value:     cneevent.ACQUIRING_SYNC,
+						Value:     ptp.ACQUIRING_SYNC,
```
Comment (aneeshkp) [inline examples/producer/main.go:97]: here too event type is not set to right enum ,  it is set to event.Type = "ptp_status_type"

**Comment by aneeshkp** [comment]
add type to this function, pick this value /sync/ptp-status/lock-state 
https://github.com/redhat-cne/cloud-event-proxy/blob/c8653d692d5885176313076f68ff5ba35de77ced/plugins/ptp_operator/metrics/metrics.go#L542

File: plugins/ptp_operator/metrics/metrics.go [hunk 2]
```diff
@@ -420,9 +419,9 @@ func (p *PTPEventManager) ExtractMetrics(msg string) {
 						log.Infof("interface %s is not anymore faulty, cancel holdover", iface)
 						t.SafeClose() // close any holdover go routines
 					}
-					p.PublishEvent(ceevent.FREERUN, ptpStats[ifaceType].LastOffset(), iface, channel.PTPEvent)
-					UpdateSyncStateMetrics(ptpStats[ifaceType].ProcessName(), iface, ceevent.FREERUN)
-					ptpStats[ifaceType].SetLastSyncState(ceevent.FREERUN)
+					p.PublishEvent(ptp.FREERUN, ptpStats[ifaceType].LastOffset(), iface, ptp.EVENT)
```
Comment (aneeshkp) [inline plugins/ptp_operator/metrics/metrics.go:422]: ptp.Event will be ptp.EventType

File: plugins/ptp_operator/ptp_operator_plugin.go [hunk 3]
```diff
@@ -233,7 +232,7 @@ func Start(wg *sync.WaitGroup, configuration *common.SCConfiguration, fn func(e
 	onReceiveOverrideFn := func(e v2.Event, d *channel.DataChan) error {
 		log.Info("got status check call,fire events for above publisher")
 		if len(eventManager.Stats) == 0 {
-			eventManager.PublishEvent(event.FREERUN, 0, "ptp-not-set", "PTP_STATUS")
+			eventManager.PublishEvent(ptp.FREERUN, 0, "ptp-not-set", "PTP_STATUS")
```
Comment (aneeshkp) [inline plugins/ptp_operator/ptp_operator_plugin.go:235]: instead of PTP_STATUS  EventPtpStateChange

Comment (jzding) [reply plugins/ptp_operator/ptp_operator_plugin.go:235]: done

File: examples/producer/main.go (hunk 1 above)
Comment (jzding) [reply examples/producer/main.go:97]: done

File: plugins/ptp_operator/metrics/metrics.go (hunk 2 above)
Comment (jzding) [reply plugins/ptp_operator/metrics/metrics.go]: done

File: README.md [hunk 4]
```diff
@@ -208,7 +208,7 @@ import (
 // create an event
 event := v1event.CloudNativeEvent()
 event.SetID(pub.ID)
-event.Type = "ptp_status_type"
+event.Type = "event.sync.ptp-status.ptp-state-change"
```
Comment (aneeshkp) [inline README.md:211]: could you use the enum here

Comment (jzding) [reply README.md]: done

**Pull request merged**